pkg/apis/workflow/v1alpha1/workflow_types.go

```diff
@@ -2118,6 +2118,9 @@ type OSSBucket struct {
 
 	// CreateBucketIfNotPresent tells the driver to attempt to create the OSS bucket for output artifacts, if it doesn't exist
 	CreateBucketIfNotPresent bool `json:"createBucketIfNotPresent,omitempty" protobuf:"varint,5,opt,name=createBucketIfNotPresent"`
+
+	// SecurityToken is the user's temporary security token
```

Comment:
could we add a link to documentation please?

Reply:
Sure. Added.